l3_dump.py

```diff
@@ -227,8 +226,26 @@ def main():
     """
     Shell to call do_main() with command-line arguments.
     """
-    check_usage()
-    (program_bin, decode_loc_id, loc_decoder_bin) = set_decode_loc_id()
+    do_main(sys.argv[1:])
+
+###############################################################################
+def do_main(args:list, return_logentry_lists:bool = False) -> int:
```

Comment:
For end-user invocation, `return_logentry_lists` will be at its default, i.e. False, so no output-lists decoding will be done.